galactic/distribution.yaml

```diff
@@ -2107,6 +2107,15 @@ repositories:
         release: release/galactic/{package}/{version}
       url: https://github.com/ros-gbp/ompl-release.git
       version: 1.5.2-1
+  open_vins:
+    doc:
+      type: git
+      url: https://github.com/rpng/open_vins
+      version: master
+    source:
+      type: git
+      url: https://github.com/rpng/open_vins
```

Comment:
This should be the `git` repo url: `https://github.com/rpng/open_vins.git` - same everywhere else `url` is mentioned in this PR.